Admin/assets/firebase/store-details.js

```diff
@@ -104,6 +110,18 @@ async function storeBillDetails(weeks,storeUID){
     updateDataTable(billDetailsData);
 }
 
+function updateBillStatus(storeUID, weekNum, btnID) {
+    console.log(storeUID, weekNum, btnID)
+    firebase.database().ref(`Teqmo/Stores/${storeUID}/payment/weeks/${weekNum}`).update({
+        'billStatus': 2 //Paid
+    }).then(() => {
+        document.getElementById(btnID).innerHTML = `Paid <i class="tio-done mr-1"></i>`
+        document.getElementById(btnID + 'Status').innerHTML = `<span class="badge badge-soft-success">
+        <span class="legend-indicator bg-success"></span>Paid
+        </span>`
+    });
```

Comment:
What if some error occurs while updating the value (very rare case but is possible). Should we consider catching and displaying the errors to the user (simple alerts will be fine)

Reply:
Added code for catching error & displaying via sweet alert